optuna/visualization/matplotlib/_pareto_front.py

```diff
@@ -85,18 +87,6 @@ def objective(trial):
 
     Returns:
         A :class:`matplotlib.axes.Axes` object.
-
-    Raises:
-        :exc:`ValueError`:
-            If ``targets`` is :obj:`None` when your objective studies have more than 3 objectives.
```

Comment:
This explanation must be added in the document.

Reply:
I think it is explained in https://github.com/optuna/optuna/pull/3429/files#diff-460cafb5bed38c8c07678e528c48960b89e0f56f5dc156936824c4a2f7436140R80-R81. Do you think there is other information that should be added?

Reply:
@HideakiImamura told me that we need to  note it (for many-objective case).

Reply:
We discussed this privately and here is more information to add.

Reply:
[8e9bee9](https://github.com/optuna/optuna/pull/3429/commits/8e9bee96e8c1ff7002fff15d7a9b1164cd6f1a64)